Libraries/Animated/src/__tests__/AnimatedNative-test.js

```diff
@@ -642,19 +642,6 @@ describe('Native Animated', () => {
       }
     });
 
-    it('fails for unsupported styles', () => {
-      const left = new Animated.Value(0);
-
-      TestRenderer.create(<Animated.View style={{left}} />);
-
-      const animation = Animated.timing(left, {
-        toValue: 10,
-        duration: 50,
-        useNativeDriver: true,
-      });
-      expect(animation.start).toThrowError(/left/);
-    });
```

Comment:
can we turn this into an affirmative test that animating these properties works now?

Reply:
Done

Reply:
Done